README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository is here to help you quickly bootstrap and use your own OpenFisca country package.
 
-## Bootstrapping a country package
+## Bootstrapping Your Country Package
```

Comment:
Ok for the Country Package, but `Your` should probably be `your` (to be consistent with `the` in `Writing the legislation`)